lib/OpenQA/WebAPI/Controller/API/V1/Job.pm

```diff
@@ -687,7 +687,9 @@ sub done {
 
 =item restart()
 
-Restart job or jobs. Used for both apiv1_restart and apiv1_restart_jobs
+Restart job(s). Use force=1 to force the restart (e.g. despite missing assets). Use skip_parents=1 to prevent restarting parent jobs.
```

Comment:
break lines

As this doc string will be shown on API help pages as well this should suffice as documentation but can we mention the difference for "directly-chained" and "not-directly-chained" dependencies here?

Reply:
I could add a line break here but it will not be shown in the API help page which doesn't seem to support any kind of formatting. This limitation is also why I would not go into any details here.